libs/datasets/combined_datasets.py

```diff
@@ -261,6 +261,11 @@ def datasource_regions(
         HHSHospitalCountyDataset,
         HHSHospitalStateDataset,
     ],
+    CommonFields.STAFFED_BEDS: [
+        CANScraperStateProviders,
```

Comment:
TBH I'm not 100% sure if we want to use CANScraperStateProviders, but I just made this match ICU_BEDS above. Chances are HHS covers all states anyway, so it'll never get used.

Reply:
I think we removed most (or all) of our non-vaccine related state level scrapers, so I'm not 100% sure that there would be state-provided data to fall back on (I could check the database to verify that), but I do think it's good to leave in